### DISCUSSION TITLE
Eachen ST-DC4 Relay Board update now not working

### DISCUSSION THREAD

**Discussion by Wheemer** [arendst/Tasmota#19163] [Q&A]
Update to latest version now relays can't be controlled.

Was on 12.0.0.2 I believe and just updated to the latest 13.0.0.2

Downgrading doesn't fix it. New flash to 13.0.0.2 doesn't solve either.

Any help appreciated.

Thank You

**Comment by barbudor** [comment]
You don't say how you upgraded so I will guess you have reflashed the device so the configuration has been erased
Follow instructions from https://templates.blakadder.com/eachen_ST-DC4.html to reconfigure it

In any case, reconfiguring should bring everything back

**Reply by Wheemer** [reply]
I did reconfigure them as per instructions. Never got a click again. I've taken it out of production for now. It has always been very unreliable. Losing locking settings every now and then. It's not worth the hassle. I replaced it with an esp32 wired to a generic relay board and things are rock solid and extremely simple now. Thank You.
